test/cli-validator/tests/info-and-hint.test.js

```diff
@@ -32,7 +32,7 @@ describe('test info and hint rules - OAS3', function() {
     consoleSpy.mockRestore();
 
     // errors for non-unique operation ids
-    expect(jsonOutput['errors']['spectral'].length).toBe(2);
+    expect(jsonOutput['errors']['spectral'].length).toBe(1);
```

Comment:
Why did all of this stuff change (including the option-handling and enabled-rules tests)?

Reply:
This changed because the Spectral, `operation-operationIds-unique` check, changed to not error on the first instance of a duplicate operation ID.

Reply:
Ah right, I forgot you updated the Spectral version. Carry on then 🙂